packages/corvid-cli/test/it/error-reporting.spec.js

```diff
@@ -63,7 +63,8 @@ describe("santy tests for error reporting", () => {
     });
   });
 
-  it("should report clone errors to sentry", async () => {
+  // eslint-disable-next-line jest/no-disabled-tests
+  it.skip("should report clone errors to sentry", async () => {
```

Comment:
change the test by mocking an error returning from the listUserSites endpoint and expecting this error to be reported